data/stories.json

```diff
@@ -1,431 +1,437 @@
+        "author": {
+            "first_name": "Tanaz",
+            "last_name": "Bhathena",
+            "origin_country": "India"
+        },
+        "video_url": "https://www.youtube.com/watch?v=ao6jHx27gB8",
```

Comment:
can you change this to the embed link (take the parameter v and place it in the url path)
i.e https://www.youtube.com/watch?v=ao6jHx27gB8 -> https://www.youtube.com/embed/ao6jHx27gB8

Reply:
Also please do this for the other youtube links

Reply:
Hmm should we validate for this? Like say the video URL must follow a certain regex or the POST is not accepted?

Reply:
We cant expect shoe project to do this b/c its kind of annoying to get this url. Instead given a regular youtube url we can convert it to the embed version by extracting the query parameter value for `v`. I think that should be a task for the story upload form (on the backend)